integration_testing/scripts/run_kolibri_app_mode.py

```diff
@@ -29,6 +32,10 @@ def RUN(self):
 
 # activate app mode
 enable_plugin("kolibri.plugins.app")
+atexit.register(disable_plugin, "kolibri.plugins.app")
```

Comment:
> and do content import actions

Is this just disabling the web interface? I couldn't login to do the above. Maybe I should try a fresh install that isn't provisioned as SoUD?

![Screenshot from 2023-03-03 10-29-04](https://user-images.githubusercontent.com/819838/222799283-b7212678-12d4-455a-9d6b-d51890770041.png)